rr_viz/src/ross/esm/environmental_sensing_module.py

```diff
@@ -69,19 +69,19 @@ def __init__(self, parent):
         self.vis_topic_sub = rospy.Subscriber(self.vis_topic_name, Illuminance, self.vis_update)
 
     def ir_update(self, msg):
-        self.ir_data_label.setText("IR: " + format(msg.illuminance, ".2f"))
+        self.ir_data_label.setText("IR: " + format(msg.illuminance, ".2f") + " Lux")
 
     def humidity_update(self, msg):
-        self.humidity_data_label.setText("Humidity: " + format(msg.relative_humidity, ".2f"))
+        self.humidity_data_label.setText("Humidity: " + format(msg.relative_humidity, ".2f") + "%")
 
     def pressure_update(self, msg):
-        self.pressure_data_label.setText("Pressure: " + format(msg.fluid_pressure, ".2f"))
+        self.pressure_data_label.setText("Pressure: " + format(msg.fluid_pressure, ".2f") + " Pascals")
 
     def temp_update(self, msg):
-        self.temp_data_label.setText("Temp: " + format(msg.temperature, ".2f"))
+        self.temp_data_label.setText("Temp: " + format(msg.temperature, ".2f") + " degC")
```

Comment:
Do you want to try your lack with using `°C` instead? There is 50/50 chance it will work

Reply:
Haha I'll try it tomorrow and if it works I'll change it!